stories/bar/bar.stories.js

```diff
@@ -43,8 +43,8 @@ export const Default = () => `
 <p><b>Compact bar with compact elements</b></p>
 <div class="fd-bar">
     <div class="fd-bar__left">
-        <div class="fd-bar__element">
-            <span class="fd-bar__element fd-bar__element--title" aria-label="text">TEXT</span>
+        <div class="fd-bar__element fd-bar__element--title">
+            <span aria-label="text">TEXT</span>
```

Comment:
Hi @sKudum the padding for Bar component changes ONLY when Title control is the first control. It won't work with a span. I mean it will, but it's not what the specs recommend. Please check the code snippets I provided in the prev review. You need to use Title component in the documentation.
<img width="598" alt="Screen Shot 2021-07-07 at 10 22 24 AM" src="https://user-images.githubusercontent.com/39598672/124776194-3e6f9100-df0d-11eb-87ff-fcdf81ca7a83.png">

Reply:
Hi @InnaAtanasova ,

 Is it mandatory to have fd-bar__element--title/fd-bar__title classes?  

We can check for title component as  first child as below and add the padding for it. Then won't be breaking change too.

```
  &__element {
      =========
     > .fd-title:first-child {
      @include fd-set-paddings-x(0.5rem,0);
    }
  }
```

With <h> tags
        ```
<div class="fd-bar__element">
            <h4 class="fd-title" aria-label="text">TEXT</h4>
        </div>
```

With <span> tags

        ```
<div class="fd-bar__element">
            <span class="fd-title" aria-label="text">TEXT</span>
        </div>
```

Observation : Creating to `fd-bar__title` padding value display is not shown as expected.

Reply:
You can't do this:
```
  &__element {
      =========
     > .fd-title:first-child {
      @include fd-set-paddings-x(0.5rem,0);
    }
  }
```
as it's violating the self contained rules. You are modifying one component inside another component.

Reply:
Sure @InnaAtanasova , 

   In this case am updating documentation example and breaking changes details as well in wikki for PR #2480 (where fd-bar__element--title class is introduced.).

<img width="609" alt="2021-07-09_13-39-11" src="https://user-images.githubusercontent.com/53487468/125046535-bb406d80-e0bb-11eb-8111-3f4438d01434.png">